src/chart/SunburstChart.tsx

```diff
@@ -102,6 +102,7 @@ function getTooltipEntrySettings({
       stroke,
       strokeWidth: undefined,
       fill,
+      nameKey: undefined,
```

Comment:
Tooltip has a title, and then `name: value` pairs. So if it shows `uv: 400` then `name = uv` and `value = 400`. dataKey decides on the value and nameKey decides on the name.

On some charts anyway. This is not consistent but I think we should make it consistent. Now with all charts going through the same codepath (redux selector) it's easy.

Reply:
Here are some tests, Funnel has the best support: https://github.com/recharts/recharts/blob/3.x/test/component/Tooltip/Tooltip.payload.spec.tsx#L179

There's a combination of `dataKey` and `name` and `nameKey` props all kinda of doing the same thing but with different scope.

Reply:
Weird. I thought a series (Line, Bar, etc) were given a name. That name now becomes the label instead of the data key. And that's it. 

nameKey seems unnecessary to me but I still need to check it out more

Reply:
Well that's correct, Line and Bar do not support nameKey.

In Funnel, name is always static, nameKey pulls from the data.

In Pie, the nameKey is always "name" and cannot be changed. Radar I think is the same.

It's a mess is what I'm trying to say.

Reply:
Whack.